test/common/testlib.py

```diff
@@ -1370,15 +1370,15 @@ def is_devel_build(self):
     def disable_preload(self, *packages):
         for pkg in packages:
             path = "/usr/share/cockpit/%s" % pkg
-            if self.machine.execute("if test -e %s; then echo yes; fi" % path):
+            if self.machine.file_exists(path):
```

Comment:
I don't understand the `self.machine` here. This helper is defined on our own class, so shouldn't it be `self.file_exists`?

Reply:
Duh!